extensions/ql-vscode/src/stories/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "lib": ["ES2021", "dom"],
     "jsx": "react",
     "sourceMap": true,
-    "rootDir": "..",
+    "rootDir": "../../..",
```

Comment:
I don't know about stories, but doesn't have this side effects?

Reply:
This basically says "look at the whole of `extensions/ql-vscode` when compiling. This is needed because we use stuff from `/test` inside `/src/stories`. Long term we might want to move `/src/stories` into `/test/stories`.

Reply:
👍